src/app/wallets/send-intraledger.ts

```diff
@@ -178,6 +182,11 @@ const executePaymentViaIntraledger = async ({
 
   const { amount, senderWallet, recipientWallet } = validationResult
 
+  // TODO Usd use case
+  assert(recipientWallet.currency === WalletCurrency.Btc)
```

Comment:
From these top level use-cases we should probably still be returning some sort of error instead of throw with `assert`